cmd/init.go

```diff
@@ -9,21 +9,27 @@ import (
 	"github.com/supabase/cli/internal/utils"
 )
 
-var initCmd = &cobra.Command{
-	GroupID: groupLocalDev,
-	Use:     "init",
-	Short:   "Initialize a local project",
-	RunE: func(cmd *cobra.Command, args []string) error {
-		fsys := afero.NewOsFs()
-		if err := _init.Run(fsys); err != nil {
-			return err
-		}
+var (
+	createVscodeWorkspace string
```

Comment:
```suggestion
	createVscodeWorkspace = new(bool)
```

Reply:
Didn't even know this was possible, updated.